app/assets/javascripts/search.js

```diff
@@ -1,114 +1,121 @@
-(function() {
-  "use strict";
+(function () {
+  'use strict'
 
-  window.GOVUK = window.GOVUK || {};
+  window.GOVUK = window.GOVUK || {}
 
-  var $ = window.jQuery;
+  var $ = window.jQuery
 
   var search = {
     init: function () {
-      var $searchResults = $('#results .results-list');
+      var $searchResults = $('#results .results-list')
 
-      search.enableLiveSearchCheckbox($searchResults);
-      search.trackSearchClicks($searchResults);
-      search.trackSearchResultsAndSuggestions($searchResults);
-      search.trackSearchResultsAndSuggestionsOnPageTrack($searchResults);
+      search.enableLiveSearchCheckbox($searchResults)
+      search.trackSearchClicks($searchResults)
+      search.trackSearchResultsAndSuggestions($searchResults)
+      search.trackSearchResultsAndSuggestionsOnPageTrack($searchResults)
     },
     buildSearchResultsData: function ($searchResults) {
-      var searchResultData = {'urls': []},
-          searchURLs = search.extractSearchURLs($searchResults),
-          searchSuggestion = search.extractSearchSuggestion();
+      var searchResultData = { 'urls': [] }
+
```

Comment:
Nitpick: we probably don't need a newline between each variable declaration.

Reply:
See https://github.com/alphagov/finder-frontend/pull/1086#discussion_r280335766